### PR TITLE
conditional on label action

### LINKED ISSUE #196: Fix labeler error for forks
The labeler action does not work on forks now https://github.com/actions/labeler/issues/12
 
An if condition could be used to check if this pr is on this repo for now: 
https://help.github.com/en/articles/contexts-and-expression-syntax-for-github-actions

### PR DESCRIPTION
**Pull request by DavidM42**
To only trigger on this specific repo
fixes #196

### CODE REVIEW

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -8,5 +8,6 @@ jobs:
 
     steps:
     - uses: actions/labeler@v2
+      if: ${{ github.repository == 'NiklasMerz/cordova-plugin-fingerprint-aio' }}
```
Comment (NiklasMerz) [inline .github/workflows/label.yml]: That should work

```suggestion
      if: contains(github.repository, "NiklasMerz/cordova-plugin-fingerprint-aio")
```

File: .github/workflows/label.yml [hunk 2]
```diff
@@ -8,5 +8,6 @@ jobs:
 
     steps:
     - uses: actions/labeler@v2
+      if: contains(github.repository, "NiklasMerz/cordova-plugin-fingerprint-aio")
```
Comment (NiklasMerz) [inline .github/workflows/label.yml]:
```suggestion
      if: contains(github.repository, 'NiklasMerz/cordova-plugin-fingerprint-aio')
```

Just those are supported? Strange

Comment (NiklasMerz) [reply .github/workflows/label.yml]: Sorry

**Comment by DavidM42** [comment]
Nice, works now

**Comment by NiklasMerz** [comment]
Thanks

**Pull request merged**